nautobot/dcim/forms.py

```diff
@@ -1990,10 +1992,19 @@ class Meta:
             "platform",
             "serial",
             "rack",
+            "position",
+            "face",
             "rack_group",
             "secrets_group",
         ]
 
+    def __init__(self, *args, **kwrags):
+        super().__init__(*args, **kwrags)
+
+        # Disable position and face because only setting null values for these fields is needed
+        self.fields["position"].disabled = True
+        self.fields["face"].disabled = True
```

Comment:
Could we allow bulk edit of both? While it doesn't address the issue as reported it could be beneficial:

- I want to bulk move devices to one face side of a rack (accidentally have them facing the hot isle instead of the cold isle)
- I want to bulk move a set of devices to a position in a rack but they are in different racks (would require not setting the rack itself in the bulk edit).

Setting rack, position, and face all in one submission would be a physical impossibility and unintended.

Reply:
> I want to bulk move a set of devices to a position in a rack but they are in different racks (would require not setting the rack itself in the bulk edit).

The endpoint used in getting rack positions requires a rack(Which means it would require setting a rack)